packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/current-pregnancy.component.tsx

```diff
@@ -189,7 +189,7 @@ const CurrentPregnancy: React.FC<PatientChartProps> = ({ patientUuid }) => {
     });
 
     return items;
-  }, [relatives, relativeToIdentifierMap, infantOutcomes]);
+  }, [relatives, patientUuid, relativeToIdentifierMap, infantOutcomes]);
```

Comment:
why are you adding `patientUuid` here?

Reply:
There's caching of this value so we want to force a refresh if the patientUuid changes